test/test_mode_app.c

```diff
@@ -0,0 +1,87 @@
+/* Wirepas Oy licensed under Apache License, Version 2.0
+ *
+ * See file LICENSE for full license details.
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "wpc.h"
+#include "platform.h"
+#include "test_mode.h"
+
+#define LOG_MODULE_NAME "test_mode_app"
+#define MAX_LOG_LEVEL INFO_LOG_LEVEL
+#include "logger.h"
+
+// Default serial port
+static char * port_name = "/dev/ttyUSB0";
+
+int main(int argc, char * argv[])
+{
+    unsigned long bitrate = DEFAULT_BITRATE;
+    LOGI("## Running test mode tests, requires FW with test-library\n");
+    if (argc > 1)
+    {
+        port_name = argv[1];
+    }
+
+    if (argc > 2)
+    {
+        bitrate = strtoul(argv[2], NULL, 0);
+    }
+
+    if (WPC_initialize(port_name, bitrate) != APP_RES_OK)
+        return -1;
+
+#if TXNODE
+    LOGI("Transmitting...\n");
+    if (Set_Test_Mode() != true)
```

Comment:
Why not just if (!Set_Test_Mode()) ?

And same in following tests.

Of course it is the same on functional point of view ;-)